app/src/main/java/com/tohami/newsapi/ui/news/ArticleFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.tohami.newsapi.ui.news;
```

Comment:
Should have a news details package with the following classes:
Fragment
view model
repo
viewmodel factory

Reply:
done